libs/designer-ui/src/lib/floatingactionmenu/helper.ts

```diff
@@ -0,0 +1,59 @@
+import type { FloatingActionMenuItem } from '.';
+import { DynamicallyAddedParameterType } from '.';
+import { getIntl } from '@microsoft/intl-logic-apps';
+
+export function getMenuItemsForDynamicAddedParameters(supportedTypes: any): FloatingActionMenuItem[] {
+  const intl = getIntl();
+  const supportedTypesArray = supportedTypes as Array<string>;
+
+  const menuItems: FloatingActionMenuItem[] = [];
+  if (supportedTypesArray.includes('text')) {
+    menuItems.push({
+      label: intl.formatMessage({ defaultMessage: 'Text', description: 'Dynamically added parameter type Text' }),
+      id: DynamicallyAddedParameterType.Text,
+      icon: 'data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+Cjxzdmcgd2lkdGg9IjQ0cHgiIGhlaWdodD0iNDRweCIgdmlld0JveD0iMCAwIDQ0IDQ0IiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPgogICAgPHRpdGxlPlRleHRfaWNvbjwvdGl0bGU+CiAgICA8ZGVmcz48L2RlZnM+CiAgICA8ZyBpZD0iRmluYWwiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiPgogICAgICAgIDxnIGlkPSJBZGQtZmlsZS0vLUVudW0tLShtYWtlcikiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0yMi4wMDAwMDAsIC0zNTMuMDAwMDAwKSI+CiAgICAgICAgICAgIDxnIGlkPSIyIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxMC4wMDAwMDAsIDI2MC4wMDAwMDApIj4KICAgICAgICAgICAgICAgIDxnIGlkPSJUZXh0X2ljb24iIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEyLjAwMDAwMCwgOTMuMDAwMDAwKSI+CiAgICAgICAgICAgICAgICAgICAgPGcgaWQ9InRleHQiPgogICAgICAgICAgICAgICAgICAgICAgICA8Y2lyY2xlIGlkPSJPdmFsLTU1NC1Db3B5IiBmaWxsPSIjOUY2Q0Q5IiBjeD0iMjIiIGN5PSIyMiIgcj0iMjIiPjwvY2lyY2xlPgogICAgICAgICAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMjUuMjAwNjYxMywxMy43NSBMMjAuMzI1NTA2MywyOC4zNDU2MjkzIEwxNy41ODgyOTMsMjAuMTQ0MjE4OCBMMTYuMjQ5OTQ4MywyMC4xNDQyMTg4IEwxMi44MzMzMzMzLDMwLjM3Mjc1MjYgTDE0LjE4MTkwNzQsMzAuMzcyNzUyNiBMMTUuMDMxNzk4OSwyNy44MTYyNTg1IEwxOC44MDY0NDI0LDI3LjgxNjI1ODUgTDE5LjY1NjMzNCwzMC4zNzI3NTI2IEwyMS4wMDQ5MDgsMzAuMzcyNzUyNiBMMjIuNzAzODM4NiwyNS4yNTgwNTk1IEwyOS4wMzY2ODExLDI1LjI1ODA1OTUgTDMwLjczNDc1OTIsMzAuMzcyNzUyNiBMMzIuMDgzMzMzMywzMC4zNzI3NTI2IEwyNi41Mzk4NTg0LDEzLjc1IEwyNS4yMDA2NjEzLDEzLjc1IFogTTI1Ljg2OTgzMzYsMTUuNzc4ODI4MyBMMjguNjA3MDQ2OSwyMy45ODAyMzg3IEwyMy4xMzI2MjA0LDIzLjk4MDIzODcgTDI1Ljg2OTgzMzYsMTUuNzc4ODI4MyBaIE0xNi45MTkxMjA3LDIyLjE3MTM0MjIgTDE4LjM3NzY2MDcsMjYuNTM2NzMyOCBMMTUuNDYwNTgwNywyNi41MzY3MzI4IEwxNi45MTkxMjA3LDIyLjE3MTM0MjIgWiIgaWQ9IlBhZ2UtMSIgZmlsbD0iI0ZGRkZGRiI+PC9wYXRoPgogICAgICAgICAgICAgICAgICAgIDwvZz4KICAgICAgICAgICAgICAgIDwvZz4KICAgICAgICAgICAgPC9nPgogICAgICAgIDwvZz4KICAgIDwvZz4KPC9zdmc+',
```

Comment:
These are generated strings copied from BPMUX. I'm not thrilled about having these as strings, but the code seems to be sensitive to how the svg string is generated, and doing the .svg -> string conversion in this file in this repo wasn't having the icons render correctly. So copied the generated strings from BPMUX